manifests/e/ElectronicArts/EADesktop/13.63.0.5576/ElectronicArts.EADesktop.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ElectronicArts.EADesktop
+PackageVersion: 13.63.0.5576
+Platform:
+- Windows.Desktop
+InstallerType: burn
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: "{5cb72f10-205d-496e-a2bc-ff98485b0f8d}"
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```

> ## Who can download the EA app?
>
> The EA app is available to all players on a (minimum) 64-bit PC system, running Windows 7, 8, 10, and 11.

According to the "FAQ" section in the package URL, this installer file is a 64-bit wizard.